browse/controllers/list_page/__init__.py

```diff
@@ -51,17 +51,17 @@
 
 from browse.controllers.abs_page import truncate_author_list_size
 from browse.controllers.list_page.paging import paging
-from browse.domain.metadata import DocMetadata
+from arxiv.document.metadata import DocMetadata
 from browse.formatting.search_authors import (AuthorList, queries_for_authors,
                                               split_long_author_list)
 from browse.services.documents import get_doc_service
-from browse.services.documents.format_codes import formats_from_source_flag
+from arxiv.formats import formats_from_source_flag
 from browse.services.listing import (Listing, ListingNew, NotModifiedResponse,
                                      get_listing_service, ListingItem)
 
 from browse.formatting.latexml import get_latexml_url
 
-from flask import request, url_for, redirect
+from flask import request, url_for, redirect, current_app
```

Comment:
`current_app` is added to the imports but I don't see any other code changes, is it not used?